Cargo.lock

```diff
@@ -2743,7 +2743,7 @@ dependencies = [
 
 [[package]]
 name = "neard"
-version = "1.1.0"
+version = "1.2.0"
```

Comment:
This is automated, but i think this should be updated, neard/Cargo.toml is 1.2.0 right now